### ISSUE TITLE
Moving event results in wrong time being displayed in email

### ISSUE THREAD

**Issue by Sarsum** [JM-Lemmi/ical-relay#274]
When changing the date of an event, the notifier displays the wrong time (timezone offset) in the generated output

**Comment by JM-Lemmi** [comment]
Duplicate of #267?

**Comment by Sarsum** [comment]
I dont think so. I believe the time is being shown correctly for upstream events, but when adding events through the UI the time is being converted into UTC and the timezone ignored by the library we use. It is correctly displayed in the web due to time conversions but we have no way to convert it into the "correct" time for the emails / webhook notifications.
